contracts/Imports.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.4.24;
+
+import "@aragon/os/contracts/kernel/Kernel.sol";
+import "@aragon/os/contracts/acl/ACL.sol";
+import "@aragon/os/contracts/factory/DAOFactory.sol";
+import "@aragon/os/contracts/factory/EVMScriptRegistryFactory.sol";
+
+contract Imports {
+
+    constructor() public {
+
+    }
```

Comment:
Not big on Solidity, but is this constructor needed? I think I understand that there needs to be a contract in here, but does it need an actual implementation?

Reply:
Nah, just my auto-fill when creating a new Solidity contract, it can be removed.

Reply:
Would be nice to have it removed for cleanliness